source/fuzz/fuzzer_util.h

```diff
@@ -483,6 +483,13 @@ std::map<uint32_t, uint32_t> RepeatedUInt32PairToMap(
 google::protobuf::RepeatedPtrField<protobufs::UInt32Pair>
 MapToRepeatedUInt32Pair(const std::map<uint32_t, uint32_t>& data);
 
+// Get the last instruction in the |block_id| that can be used to insert
+// |opcode| above itself. Returns nullptr if no such an instruction is present
+// in the block.
```

Comment:
Maybe:

Returns the last instruction in |block_id| before which an instruction with opcode |opcode| can be inserted, or nullptr if there is no such instruction.

Reply:
Fixed in bcb5394.